.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.4.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint 
+        run: |
+          npm install --save-dev hint@6.0.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/gh-actions/html-css/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint --telemetry=off .
```

Comment:
- [ ] Great job setting up your linters file, you are almost there! However, there's still an error showing up in the webhint test:

![Captura de pantalla 2022-04-19 a la(s) 11 57 38 p  m](https://user-images.githubusercontent.com/83717285/164153766-6f966635-e636-4299-a0b0-0016f38ab21b.png)

- Your linters file has a different command on line 34, please make sure to copy this [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) this will probably help you to pass this test, once you get all three tests green submit your PR again for review and you will be done! 

| Original | Yours |
|---|---|
| ![Captura de pantalla 2022-04-20 a la(s) 12 00 57 a  m](https://user-images.githubusercontent.com/83717285/164153961-550d3c1b-18a0-4c00-b8fe-6c8c4a9e48b9.png) | ![Captura de pantalla 2022-04-20 a la(s) 12 00 48 a  m](https://user-images.githubusercontent.com/83717285/164154003-ae8f8bb3-35fc-4304-a4d1-499f666c15c9.png)|